src/containers/ToDoList/ToDoList.jsx

```diff
@@ -54,6 +58,23 @@ export const ToDoList = () => {
     }
   };
 
+  const handleClearCompletedTasks = () => {
+    setTasks(tasks.filter((task) => !task.checked));
```

Comment:
You're doing this `(task) => !task.checked)` couple of times in the code.

While above the component itself you could attach these functions to variables, like:

```
const activeTasksFilter = (task) => !task.checked
const completedTasksFilter = (task) => task.checked)
```

and then just reuse these vars like:
```
setTasks(tasks.filter(completedTasksFilter)
```